SPRINTUSER_README.md

```diff
@@ -50,19 +50,25 @@ Find the Docker installer for your Operating System underneath the **installs**
  MacOS | Docker CE | Docker.dmg
  Linux | Docker CE, docker-compose | See [Linux instructions](https://docs.docker.com/install/#docker-ce)
 
-**⚠️ Docker Toolbox users**: Please install "Git for Windows" and "VirtualBox" as needed during the install process.
+**⚠️Docker Toolbox users**: Please click the checkboxes to install "Git for Windows" and "VirtualBox" as needed during the install process.
 
-**⚠️ Docker CE only** Start Docker
+**⚠️Docker Toolbox Users must start "Docker Quickstart Terminal" to start docker running.
 
-**⚠️ Docker CE only** Open Docker preferences and set docker memory allocation to 3.0GB or greater in the Advanced section. This is required to start containers.
```

Comment:
Isn't this still a good (or necessary) step?

Reply:
Tell me more - You can't share drives, but things work ok with docker toolbox anyway? Is that a path for people with locked-down computers or something?

Reply:
Yes, Docker Toolbox works fine even though I don't have access to File & Print Sharing, but I do have enough Admin. access to run installers.

Reply:
I suspect that you might be one of many then. We may want to explain that option to people in your position. Fascinating to know!